assets/js/blocks/active-filters/style.scss

```diff
@@ -15,17 +15,48 @@
 			background: transparent none;
 		}
 	}
-	.wc-block-active-filters-list {
+	.wc-block-active-filters__list {
 		margin: 0 0 $gap-smallest;
 		list-style: none outside;
 		clear: both;
 
 		li {
-			margin: 0 0 $gap-smallest;
-			padding: 0 16px 0 0;
+			margin: 0;
+			padding: 0;
 			list-style: none outside;
 			clear: both;
-			position: relative;
+
+			ul {
+				margin: 0;
+				padding: 0;
+				list-style: none outside;
+			}
+
+			.wc-block-active-filters__list-item-type {
+				text-transform: uppercase;
+				font-size: 0.7em;
+				letter-spacing: 0.1em;
+				margin: $gap 0 0;
+				display: block;
+			}
+
+			.wc-block-active-filters__list-item-operator {
+				font-weight: normal;
+				font-style: italic;
+			}
+
+			.wc-block-active-filters__list-item-name {
+				font-weight: bold;
+				display: block;
+				position: relative;
+				padding: 0 16px 0 0;
```

Comment:
This right padding doesn't seem to play well with the `chips` display style:

![imatge](https://user-images.githubusercontent.com/3616980/73368916-17da0f00-42b2-11ea-9670-9cca71d51da8.png)